src/index.html

```diff
@@ -1,14 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <img src="./images/logo.png" alt="logo">
+      <nav>
+        <ul class="nav">
+          <li class = "nav_item is-active"><a class = "link" href="#">Apple</a></li>
+          <li class = "nav_item"><a class = "link" href="#">Samsung</a></li>
+          <li class = "nav_item"><a class = "link" href="#"> Смартфоны и Телефоны</a></li>
+          <li class = "nav_item">
+            <a
+              class="link"
+              href="#"
+              data-qa="hover"
+              >Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class = "nav_item" ><a class = "link" href="#">Гаджеты</a></li>
```

Comment:
```suggestion
          <li class = "nav__item" ><a class = "link" href="#">Гаджеты</a></li>
```
два подчеркивания, я уже писал за это